src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's block in the address.
+ *
+ */
+public class Block {
+
+    public static final String EXAMPLE = "123";
+
+    public final String value;
+    private boolean isPrivate;
+
+    /**
+     * Constructor
+     *
+     */
+    public Block(String block, boolean isPrivate) {
```

Comment:
Would adding another validation step help improve the class?
Furthermore it is okay to not comment for constructor unless it's task is complex =P.